src/main/java/example/models/PlayerCountry.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
we don't need this entity cos the table is anyways created through changelog.xml